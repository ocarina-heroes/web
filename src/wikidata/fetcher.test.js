import fetchHeroes from './fetcher'

test('fetches heroes', async () => {
  const heroes = await fetchHeroes()
  console.info(heroes)
  expect(heroes).toMatchSnapshot()
})
