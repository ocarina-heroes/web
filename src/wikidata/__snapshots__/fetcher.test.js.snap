// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetches heroes 1`] = `
Array [
  Object {
    "dateOfBirth": "1929-01-15T00:00:00Z",
    "dateOfDeath": "1968-04-04T00:00:00Z",
    "image": "http://commons.wikimedia.org/wiki/Special:FilePath/Martin%20Luther%20King%2C%20Jr..jpg",
    "item": Object {
      "label": "Martin Luther King Jr.",
      "value": "Q8027",
    },
    "occupation": "writer/pastor/preacher/civil rights advocate/theologian/minister/human rights activist/pacifist/peace activist/humanitarian",
  },
  Object {
    "dateOfBirth": "1913-02-04T00:00:00Z",
    "dateOfDeath": "2005-10-24T00:00:00Z",
    "image": "http://commons.wikimedia.org/wiki/Special:FilePath/Rosaparks.jpg",
    "item": Object {
      "label": "Rosa Parks",
      "value": "Q41921",
    },
    "occupation": "public figure/civil rights advocate/human rights activist/autobiographer",
  },
]
`;
